### PR TITLE
Adds names & descriptions for Frontiersman and Ramzi mobs

### PR DESCRIPTION
**Pull request by Apogee-dev**
## About The Pull Request

Adds names and descriptions to the Frontiersman and Ramzi mobs, making it a little easier to recognize which mob is doing what in chatlogs and giving them a little more (possibly too much) flavor if you shift click them. I might have cooked too hard on the miniboss/elite mobs.

## Why It's Good For The Game

ramzis being named Ramzi Clique Operative regardless of whether they're punching you with the force of a sponge or turning you to mist with a Bulldog is a little annoying, and their descs are extremely minimal

## Changelog

:cl:
add: Ramzi and Frontiersman mobs now have more names and descriptions
/:cl:

### CODE REVIEW

File: code/modules/mob/living/simple_animal/hostile/human/syndicate.dm [hunk 1]
```diff
@@ -168,22 +182,26 @@
 	set_light(4)
 
 /mob/living/simple_animal/hostile/human/syndicate/ranged/space/stormtrooper
+	name = "Ramzi Clique Pistolmaster" //I can't think of something better, sue me
```
Comment (MarkSuckerberg) [inline code/modules/mob/living/simple_animal/hostile/human/syndicate.dm:185]: how could you not do GUNSLINGER, smh smh smh

**Review by MarkSuckerberg — APPROVED** [review]
love this so much honestly, super fun to see and I'd love to come across them in-game

**Comment by Sun-Soaked** [comment]
resolve confliiicts and I will merge.... (also gunslinger would probably be better than pistolmaster but you do iou)

**Comment by Apogee-dev** [comment]
Ramzis aren't really the rootin' tootin' revolver slinging gunslinger types. It didn't feel fitting.

**Review by Sun-Soaked — APPROVED** [review]

**Pull request merged**